flytekit/core/base_task.py

```diff
@@ -24,6 +24,8 @@
 from dataclasses import dataclass
 from typing import Any, Coroutine, Dict, Generic, List, Optional, OrderedDict, Tuple, Type, TypeVar, Union, cast
 
+from flyteidl.core import tasks_pb2 as _core_task
```

Comment:
let's not use the _ anymore?  i'm actively trying to remove it from the code base.

Reply:
Will clean up. Are you using `tasks_pb2` directly, or still aliasing as `core_task`?